en/guide/development-tools.md

```diff
@@ -58,7 +61,7 @@ export default {
 </style>
 ```
 
-When we launch our app with `npm run dev` and open [http://localhost:3000](http://localhost:3000), we can see our red `Hello world!` title.
+When we launch our app with `npm run dev` and open http://localhost:3000, we can see our red `Hello world!` title.
```

Comment:
Avoid link for all localhost path because in most cases that conduces to 404 page.